common/v2/services/LocalCache/constants.ts

```diff
@@ -1,153 +1,109 @@
       timestamp: Date.now()
     }
   },
-  transactionHistories: {
-    '6e1c322c-aea6-4484-8fdc-7b3227a9d359': {
-      transaction: '6e1c322c-aea6-4484-8fdc-7b3227a9d359'
-    },
-    '76b50f76-afb2-4185-ab7d-4d62c0654881': {
-      transaction: '76b50f76-afb2-4185-ab7d-4d62c0654881'
-    },
-    '76b50f76-afb2-4185-ab7d-4d62c0654885': {
-      transaction: '76b50f76-afb2-4185-ab7d-4d62c0654885'
-    }
-  },
-  transactions: {
-    '6e1c322c-aea6-4484-8fdc-7b3227a9d359': {
-      stage: 'pending',
-      label: 'Example',
-      date: 1547768373,
-      value: 0,
-      from: '0x80200997f095da94E404F7E0d581AAb1fFba9f7d',
-      to: '0x80200997f095da94E404F7E0d581AAb1fFba9f7d',
-      fiatValue: {
-        USD: '0'
-      }
-    },
-    '76b50f76-afb2-4185-ab7d-4d62c0654881': {
-      stage: 'completed',
-      label: 'Example',
-      date: 1547769373,
-      value: 0,
-      from: '0x80200997f095da94E404F7E0d581AAb1fFba9f7d',
-      to: '0x80200997f095da94E404F7E0d581AAb1fFba9f7d',
-      fiatValue: {
-        USD: '0'
-      }
-    },
-    '76b50f76-afb2-4185-ab7d-4d62c0654885': {
-      stage: 'completed',
-      label: 'Example',
-      date: 1548769373,
-      value: 0,
-      from: '0x80200997f095da94E404F7E0d581AAb1fFba9f7d',
-      to: '0x80200997f095da94E404F7E0d581AAb1fFba9f7d',
-      fiatValue: {
-        USD: '0'
-      }
-    }
-  },
-  accountTypes: {
+  wallets: {
     MetaMask: {
       name: 'MetaMask',
       key: 'metamask',
       secure: true,
       derivationPath: '',
       web3: true,
-      hardware: false
+      hardware: false,
+      desktopOnly: false
     }
   },
   assets: {
     '12d3cbf2-de3a-4050-a0c6-521592e4b85a': {
-      option: 'Ethereum',
-      amount: '14.13',
-      network: 'ETH',
+      uuid: '12d3cbf2-de3a-4050-a0c6-521592e4b85a',
+      name: 'Ethereum',
+      networkId: 'ETH',
       type: 'base',
-      symbol: 'ETH'
+      ticker: 'ETH',
+      decimal: 18,
+      contractAddress: null
     },
     '10e14757-78bb-4bb2-a17a-8333830f6698': {
-      option: 'OmiseGo',
-      amount: '2',
-      network: 'ETH',
+      uuid: '10e14757-78bb-4bb2-a17a-8333830f6698',
+      name: 'OmiseGo',
+      networkId: 'ETH',
       type: 'erc20',
-      symbol: 'OMG',
+      ticker: 'OMG',
       contractAddress: '0xd26114cd6EE289AccF82350c8d8487fedB8A0C07',
       decimal: 18
     }
   },
-  localSettings: {
-    '17ed6f49-ff23-4bef-a676-69174c266b37': {
-      fiatCurrency: 'GBP',
-      favorite: false
-    }
-  },
-  networkOptions: {
+  networks: {
     ETH: {
       id: 'ETH',
       name: 'Ethereum',
-      unit: 'ETH',
+      baseAsset: '12d3cbf2-de3a-4050-a0c6-521592e4b85a',
       chainId: 1,
       isCustom: false,
       color: '#007896',
       blockExplorer: {},
       tokenExplorer: {},
-      tokens: [],
       contracts: ['17ed6f49-ff23-4bef-a676-69174c266b38'],
-      nodes: ['eth_mycrypto'],
-      dPathFormats: {
+      nodes: [
+        {
+          name: 'eth_mycrypto',
```

Comment:
Consider changing key to id/uuid

Reply:
Actually, i think name is more accurate here. When the user creates a new node they specify a name for it, so it makes sense to keep it this way imo.

Reply:
That's fine with me, I only mentioned it because it was in the data model document